test/integration/switch_locale_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+
+class SwitchLocaleTest < ActionDispatch::IntegrationTest
+  test "I18n.locale will depends on the locale of urls" do
+    [:en, :"zh-TW"].each do |locale|
```

Comment:
```suggestion
    I18n.available_locales.each do |locale|
```

Reply:
I think a combination of the default locale and another one is enough to bring our attentions when things break.
How do you think?

Reply:
Good point.